frontend/public/components/catalog/catalog-items.tsx

```diff
@@ -125,24 +178,48 @@ const getAvailableFilters = (initialFilters) => {
     },
   };
 
+  filters.capabilityLevel = {
+    BasicInstall: {
+      value: 'Basic Install',
```

Comment:
Do these values end up in the url? They should probably be lower case be no spaces.

Reply:
yes, they do... they cannot be case changes or altered directly as they need to be an exact string match with the annotation.
```
  value: 'BasicInstall' || 'basicinstall' will not match with metadata.annotations.cabalities: Basic Install
```
Following the above recommendation, I am changing the vales to lowercase and also transforming the the capability data obtained from the CSV while adding it to the mock object 
in dev-catalog.ts, line 59